frontend/src/libs/components/button/button.tsx

```diff
@@ -1,10 +1,24 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
+  className?: string;
 };
 
-const Button: React.FC<Properties> = ({ type = 'button', label }) => (
-  <button type={type}>{label}</button>
+const Button: React.FC<Properties> = ({
+  type = 'button',
+  label,
+  className = '',
+}) => (
+  <button
+    type={type}
+    className={getValidClassNames(styles['button'], className)}
```

Comment:
After discussion with coaches we decided to get rid of className form props. The reason why is passing styles like margin to button. This is not correct approach) We need to wrap for such purpose

Reply:
So how it should be like. I should wrap button in its component or wrap in component, were it is used?

Reply:
@v0ldemar01 In issue #49 I have to massivly change button styles (background-color, color, border-radius e.t.c.). If there's no className prop, can you explain how is it supposed to be done?

Reply:
We can add something like style property,  for example: 
```
style?: 'primary' | 'secondary' | 'plain';
```

Reply:
@Ekit18 for margins we should wrap button component inside div and add margin too it, we should not add margins to common components

Reply:
I understand, thanks!

Reply:
done